.github/workflows/explosionbot.yml

```
@@ -0,0 +1,28 @@
+name: Explosion Bot
+
+on:
+  issue_comment:
+    types:
+      - created
+      - edited
+
+jobs:
+  explosion-bot:
+    if: github.repository_owner == 'explosion'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Dump GitHub context
+        env:
+          GITHUB_CONTEXT: ${{ toJson(github) }}
+        run: echo "$GITHUB_CONTEXT"
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+      - name: Install and run explosion-bot
+        run: |
+          pip install git+https://${{ secrets.EXPLOSIONBOT_TOKEN }}@github.com/explosion/explosion-bot
+          python -m explosionbot
+        env:
+          INPUT_TOKEN: ${{ secrets.EXPLOSIONBOT_TOKEN }}
+          INPUT_BK_TOKEN: ${{ secrets.BUILDKITE_SECRET }}
+          ENABLED_COMMANDS: "test_llm_gpu,test_llm_external"
```

```suggestion
          ENABLED_COMMANDS: "test_gpu,test_llm_external"
```
On our explosion bot repo, it seems like we want to reuse the existing spaCy command `test_gpu` - there is no new definition for `test_llm_gpu` like there is for `test_llm_external`. I'll merge with this edit, so we can start testing this properly.